nodes/src/nodes/tool_git/git_repo.py

```
@@ -0,0 +1,952 @@
+            continue
+        if obj.type_str == 'tree':
+            # Recurse if this directory could contain files that satisfy the prefix.
+            if not prefix or _path_matches(entry_path, prefix) or prefix.startswith(entry_path + '/'):
+                _grep_tree(repo, obj, rx, prefix, results, entry_path)
+        elif obj.type_str == 'blob':
+            if prefix and not _path_matches(entry_path, prefix):
+                continue
+            try:
+                text = obj.data.decode('utf-8', errors='replace')
+            except Exception:
+                continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)